user_templates/moonraker.conf

```diff
@@ -31,6 +31,13 @@
 # [include moonraker/z_calibration.conf]
 # -----------------------------------------------------------------
 
+##### Happy_Hare V2 plugin update management ----------------------
+[update_manager happy-hare]
+type: git_repo
+path: ~/Happy-Hare
+origin: https://github.com/moggieuk/Happy-Hare.git
+install_script: install.sh
+managed_services: klipper
```

Comment:
this should go to a moonraker/mmu.conf file. the current lining means MMU for everyone ;)

Reply:
You're right, we could even do without it because the HHv2 install script normally creates the entry in moonraker.conf. But I'm not completely sure with the modifications that moggieuk made to his script to detect klippain... To test (on my todo list)